### ISSUE TITLE
Color contrast of breadcrumbs nav

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-bigpicture#38]
The nav bar breadcrumbs text to bg color contrast is only 2.8:1 and should be adjusted to at least 4.5:1. The text color stems from the `--mm-color-text-dimmed` variable, which may be resulting in problematic contrast elsewhere in the theme that I haven't identified in my glance over.

**Comment by kimisgold** [comment]
I've changed it to use the default text and added a bold weight to identify it as a heading.

**Comment by aviannamiller** [comment]
Looks great!
